.pre-commit-config.yaml

```diff
@@ -15,6 +15,13 @@ repos:
     rev: v3.12.0
     hooks:
     -   id: reorder-python-imports
+-   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
FYI we in general rely on running mypy with pre-commit instead of having a dedicated job on CI. 👍